content/features/service_discovery.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Service Discovery"
+file: "service_discovery"
+date: 2018-07-18T12:33:50+02:00
+draft: false
+youTubeID: oG-WDA-hmvA
+image: "features/service_discovery.png"
+subTitle: "Service view, Pods, Deployments, Rules, Metrics..."
+description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur rutrum pharetra tristique. Vivamus nisl ipsum, interdum at accumsan nec, molestie sit amet dolor. Sed mattis eros leo, ut sagittis mauris fermentum eget. Donec ut mauris interdum, accumsan lacus id, dignissim tortor. Maecenas non risus eget dolor interdum convallis tincidunt efficitur. Etiam accumsan justo sed enim vestibulum, vel ultrices nisl pellentesque. Vestibulum quis est ex. Praesent luctus dapibus mi non commodo. Praesent in tortor euismod, finibus nulla ut, dapibus lacus. Suspendisse ac odio elit. Praesent quis ultricies justo. Vestibulum eu dapibus arcu, quis faucibus ex. Aliquam sit amet suscipit libero. Cras tempor varius orci, ac auctor nibh efficitur id."
+---
+
+# sada
+
+<center>
+  <iframe width="360" height="215" src="https://www.youtube.com/embed/oG-WDA-hmvA" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</center>  
```

Comment:
I would put this as the single.html template, similar to https://github.com/kiali/kiali.io/blob/master/layouts/api/single.html it  would be in layouts/features/single.html and applied to all single page feature. Rather than copy/pasting the inclusion code for YouTube, and repeat the YouTubeID

Reply:
PS: I am not sure if we need a whole page to describe a feature. Explaining a new feature should remain "lightweight" or I'm afraid we won't do / update them as often as needed